### DISCUSSION TITLE
Cannot grant permission to role

### DISCUSSION THREAD

**Discussion by khanhduatd186** [abpframework/abp#20202] [Q&A] [answered]
I have two modules that share the same core. The first module can still set permissions for the role, but the second module cannot. Please give me a solution.
![test1](https://github.com/abpframework/abp/assets/127516595/1feca0aa-4ef0-4c63-b29d-05bee91c339c)

**Comment by realLiangshiwei** [comment, marked as answer]
I can't help you with this information.
ABP permission module is open source, you can debug it
